lib/amplitude_web.dart

```diff
@@ -0,0 +1,46 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+import 'web/amplitude_js.dart';
+import 'dart:async';
+
+class AmplitudeFlutterPlugin {
+  static void registerWith(Registrar registrar) {
+    final channel = MethodChannel(
+      'amplitude_flutter',
+      const StandardMethodCodec(),
+      registrar,
+    );
+    final pluginInstance = AmplitudeFlutterPlugin();
+    channel.setMethodCallHandler(pluginInstance.handleMethodCall);
+  }
+
+  /// Handles method calls over the MethodChannel of this plugin.
+  /// Note: Check the "federated" architecture for a new way of doing this:
+  /// https://flutter.dev/go/federated-plugins
+  Future<dynamic> handleMethodCall(MethodCall call) async {
+    var args = jsonDecode(call.arguments.toString());
+    var instanceName = args['instanceName'];
+    Amplitude amplitude = Amplitude.getInstance(instanceName);
+    switch (call.method) {
+      case "init":
+        {
+          String apiKey = args['apiKey'];
+          return amplitude.init(apiKey, args['userId'] ?? null);
```

Comment:
nit: this default behavior could be moved out of directly calling

Reply:
fixed